test/registerTest.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
+var protobuf = require('protobufjs');
+var MetricFamily;
+protobuf.load('metrics.proto', function(err, root) {
```

Comment:
If you build it before running tests that might be cleaner. Your choice of course